src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -48,14 +48,12 @@ class CoreWorkerDirectActorTaskSubmitter {
   /// Submit a task to an actor for execution.
   ///
   /// \param[in] task The task spec to submit.
-  /// \return Status::Invalid if the task is not yet supported.
-  Status SubmitTask(TaskSpecification task_spec);
+  void SubmitTask(TaskSpecification task_spec);
 
   /// Tell this actor to exit immediately.
   ///
   /// \param[in] actor_id The actor_id of the actor to kill.
-  /// \return Status::Invalid if the actor could not be killed.
-  Status KillActor(const ActorID &actor_id);
+  void KillActor(const ActorID &actor_id);
```

Comment:
Those methods never failed so I figured it's quite useless to return `Status`, but I can change it back if you think this violates some project conventions.